vendor/engines/sanger_sequencing/app/assets/javascripts/sanger_sequencing/util.coffee

```diff
@@ -0,0 +1,8 @@
+exports = exports ? @
+exports.SangerSequencing = new Object unless exports.SangerSequencing
+
+class exports.SangerSequencing.Util
+  @flattenArray: (arrays) ->
```

Comment:
This should get exercised indirectly in other specs but I wouldn't mind seeing specs for `flattenArray` itself.